assets/css/sign_groups.css

```diff
@@ -117,3 +117,46 @@
 .sign_groups--buttons-container * {
   margin-left: 1rem;
 }
+
+.sign_groups--groups-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.sign_groups--create-button {
+  height: min-content;
+  flex: 0 0 10%;
+}
+
+.sign_groups--group-list {
+  display: flex;
+  flex: 0 0 80%;
+  max-height: 20rem;
+  flex-direction: column;
+  gap: 1rem;
+  overflow: scroll;
+}
+
+.sign_groups--group-item {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
```

Comment:
Ah, I was wondering why the vertical list needed to be flex, rather than just a series of divs with a `bottom-margin`. But I suppose this is to have an even spacing in between the items without having to do a CSS `:first-child` or `:last-child` marker to negate the spacing on the outside? It looks like it only eliminates Internet Explorer which we don't support anyway, so that's probably fine, and maybe a little cleaner.

Reply:
I got this from https://web.dev/learn/css/spacing/#grid-and-flexbox, I wasn't aware it was new enough to break IE. Seemed simple, and I appreciate not having to negate the external spacing like you mention, but I'm open to alternatives.